packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.handler.ts

```diff
@@ -120,9 +122,16 @@ async function getInfoForAllTeams({ ctx, input }: GetOptions) {
     limit,
   });
 
+  // Get total team count across all teams the user is in (for pagination)
+
+  const totalTeamMembers =
+    await prisma.$queryRaw<number>`SELECT COUNT(DISTINCT "userId")::integer from "Membership" WHERE "teamId" IN (${Prisma.join(
+      teamIds
+    )})`;
+
   return {
     teamMembers,
-    totalTeamMembers: teamMembers.length,
```

Comment:
No wonder it wasnt working for teams - teamMembers is always the length of the limit via pagination